garrysmod/lua/postprocess/dof.lua

```diff
@@ -43,14 +45,14 @@ function DOF_Start()
 
 	DOFModeHack( true )
 
-end
+	hook.Add( "Think", "DOFThink", function()
 
-hook.Add( "Think", "DOFThink", function()
+		DOF_SPACING = pp_dof_spacing:GetFloat()
+		DOF_OFFSET = pp_dof_initlength:GetFloat()
 
-	DOF_SPACING = pp_dof_spacing:GetFloat()
-	DOF_OFFSET = pp_dof_initlength:GetFloat()
+	end)

```

Comment:
Unnecessary change that does not conform to the style guidelines:
https://github.com/Facepunch/garrysmod/blob/master/CONTRIBUTING.md